src/app/components/pool-size-chart/pool-size-chart.component.ts

```diff
@@ -14,11 +12,13 @@ import { PoolSizeChartTheme } from './pool-size-chart-theme.enum';
   styleUrls: ['./pool-size-chart.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class PoolSizeChartComponent implements OnInit {
+export class PoolSizeChartComponent implements OnInit, OnChanges {
   @Input() sizes: PoolSize[];
   @Input() theme: PoolSizeChartTheme = PoolSizeChartTheme.Dark;
-
+  @Input() config: any;
```

Comment:
@Input() is for things that's passed from the outside, not needed here.
Also here and below the type is `google.visualization.LineChartOptions`.